### ISSUE TITLE
Homepage - Parsing errors that might affect accessibility 

### ISSUE THREAD

**Issue by iamjolly** [18F/samhsa-prototype#315]
### Problem Description
Using the W3C validator to check for [WCAG 4.1.1 Parsing (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/parsing), the following errors were flagged. The most important issue to fix is the first error listed.

1. `div` not allowed as child element of `label` within the treatment facility location form. 
![samhsa-w3-parsing-a11y-error-high-priority](https://user-images.githubusercontent.com/1093423/64875754-f3cfa700-d60a-11e9-9f93-5d8799ee1c6d.png)

2. `value` attribute not allowed on `label` element
![samsha-w3-parsing-error-low-priority](https://user-images.githubusercontent.com/1093423/64875755-f3cfa700-d60a-11e9-929c-566f553db166.png)

**Note:** These errors may or may not have direct impact on accessibility, as browsers often correct for errors in document syntax. However, this is still a _technical_ issue where the code does not meet the WCAG 2.0 SC for Parsing (4.1.1).
